readme/Rewrites.scalatex

```diff
@@ -219,26 +219,5 @@
     @p
       The two syntaxes are equivalent and the presence of the @code{val} keyword has been deprecated since Scala 2.10.
 
-  @sect(NoExtendsApp.toString)
```

Comment:
cc/ @gabro 

I'm going on vacation next week so I didn't have time to closely debug the failures. I suspect it's not too hard to fix, but given #215 I felt it would be best to remove the rewrite since it already had enough problems. I will try to add the rewrite back again once I'm back from vacation in August.